### ISSUE TITLE
embedding

### ISSUE THREAD

**Issue by Macielyoung** [gcooq/TUL#1]
Hello， I want to ask how to produce embedding vector ? Just use word2vec ?

**Comment by gcooq** [comment]
Thanks for your attention, you first  split the long trajectory to short trajectory with time interval. Then you can use the word2vec tookite training the datasets

**Comment by Macielyoung** [comment]
You have defined your loss function, so do you call word2vec directly ?  Your embedding vector has been trained in advance and doesn't exist in your code.

**Comment by gcooq** [comment]
I used the tool of word2vec, you can download this toolkit via python command. First pretraining all of the trajectory data by word2vec and saved these vectors, then you can use the sourcecode . I will upload the details and some about datasets soon.

**Comment by Macielyoung** [comment]
OK, thanks !

**Comment by mingzhang798** [comment]
Hello, I want to know how you concatenate the location points within six hours of the original Gowalla data into a track. Can you share the code?
